plugins/radar-android-google-places/src/main/java/org.radarbase.passive.google.places/GooglePlacesManager.kt

```diff
@@ -206,15 +210,18 @@ class GooglePlacesManager(service: GooglePlacesService, @get: Synchronized priva
 
     private fun doSend(places: List<PlaceLikelihood>) {
         places.forEach { likelihood ->
-            val types = likelihood.place.types?.map { it.toPlacesType() }
+            val types = likelihood.place.placeTypes
```

Comment:
Using Places.placeTypes, which returns a list of strings, instead of the deprecated Places.Type.